src/platform-includes/capture-error/flutter.mdx

```diff
@@ -1,4 +1,5 @@
 <PlatformContent includePath="capture-error/dart.mdx" />
 
 - Flutter-specific errors, such as using `FlutterError.onError`, are captured automatically
-- The SDK already runs your init `callback` on an error handler, such as using `runZonedGuarded`, are captured automatically
+- For Flutter <3.3, the SDK runs your init `appRunner` callback in a custom error zone, using [`runZonedGuarded`](https://api.flutter.dev/flutter/dart-async/runZonedGuarded.html) to automatically record errors not caught by flutter.
+- Starting with Flutter 3.3, the SDK uses [`PlatformDispatcher.onError`](https://api.flutter.dev/flutter/dart-ui/PlatformDispatcher/onError.html) to automatically record errors not caught by flutter.
```

Comment:
```suggestion
- The SDK already runs your init `callback` on an error handler, such as using `runZonedGuarded` on Flutter below `3.3` or `PlatformDispatcher.onError` on Flutter equal or above 3.3.
```

I don't see the need of modifying the original sentence, IMO its more confusing.

The sentence was short and concise.

We could simply go with something similar to this.

What do you think @imatwawana ?

Reply:
@denrase I'd do the same for https://github.com/getsentry/sentry-dart/pull/1082
Its even more confusing in there IMO.

Reply:
To be honest, I find the two new sentences easier to understand. There were words missing in the original version that make it so I don't understand what the original version was trying to say. _What_ is being automatically captured? That's clearer in the new versions. We can try to make it more concise though. Suggest the following:

```suggestion
The SDK already runs your init `callback` on an error handler, such as [`runZonedGuarded`](https://api.flutter.dev/flutter/dart-async/runZonedGuarded.html) on Flutter versions prior to `3.3`, or [`PlatformDispatcher.onError`](https://api.flutter.dev/flutter/dart-ui/PlatformDispatcher/onError.html) on Flutter versions 3.3 and higher, so that errors are automatically captured.
```

Reply:
> The suggestion is great, actually very similar to my suggestion [#5661 (comment)](https://github.com/getsentry/sentry-docs/pull/5661#discussion_r999008592) So +1

Indeed! I was using your version for structure and the other two versions for meaning. So definitely a group effort!